app/views/clients/show.erb

```diff
@@ -0,0 +1,47 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <!--  <a class="navbar-brand" href="#">Navbar</a>-->
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="/clients">Clients</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/products">Products</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/logout">Log Out</a>
+      </li>
+    </ul>
+  </div>
+</nav>
+<h1>Here is the Client:</h1>
+<ul>
+  <li>First Name:&nbsp;<%= @client.first_name %></li>
+  <li>Last Name:&nbsp;<%= @client.last_name %></li>
+  <li>Nickname:&nbsp;<%= @client.aka_name %></li>
+  <li>Age:&nbsp;<%= @client.age %></li>
+  <li>Gender:&nbsp;<%= @client.gender %></li>
+  <li>Needs:&nbsp;<%= @client.need %></li>
+  <% if @client.user != nil %>
+    <li>Belongs To:&nbsp;<%= @client.user.username %></li>
+  <% end %>
+</ul>
+<p>Client has the following products.</p>
+<% ClientProduct.all.each do |item| %>
```

Comment:
Here, we iterate over every ClientProduct, and for each one that is associated with the client, print information about the first product with a relationship to the client -- and only the first. How can we use `@client.products` to print info about each product?

Reply:
I thought that's what I am doing on lines 32-38?

Reply:
Okay know I see it, I am fixing it.